js/musicutils.js

```diff
@@ -286,6 +286,24 @@ var MODENAMES = [
     [_('custom'), 'custom'],
 ];
 
+var OCTAVESELECTORS = [
+    [-1,-1],
+    [0,0],
+    [1,1],
+    [_('previous'),'previous'],
```

Comment:
The second array element should be based on the current octave and returned as a number.